src/Api/Middlewares/ExceptionMiddleware.cs

```diff
@@ -82,6 +84,18 @@ private async void HandleRequestValidationException(HttpContext context, Excepti
         await context.Response.Body.WriteAsync(SerializeToUtf8BytesWeb(result));
     }
 
+    private async void HandleEntityNotAvailableException(HttpContext context, Exception ex)
+    {
+        context.Response.StatusCode = StatusCodes.Status404NotFound;
```

Comment:
Should be something other than `404 not found`
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404

Reply:
chiến said use 404

Reply:
use Confilct some thing with 409 status code and remove this shit